src/frontend/Komponenter/Journalforing/Behandling.tsx

```diff
@@ -104,29 +131,18 @@ const BehandlingInnold: React.FC<Props> = ({ behandling, settBehandling, fagsak
                                         <td>–</td>
                                     </StyledNyBehandlingRad>
                                 )}
-                            </tbody>
-                        </table>
-                        {fagsak.behandlinger.every(
-                            (behandling: Behandling) => behandling.status !== 'UTREDES'
-                        ) &&
-                            !nyBehandling && (
-                                <Flatknapp
-                                    onClick={() => {
-                                        settNyBehandling({
-                                            behandlingstype: utledRiktigBehandlingstype(
-                                                fagsak.behandlinger
-                                            ),
-                                        });
-                                    }}
-                                >
-                                    <LeggtilMedSirkel />
-                                    <span>Opprett ny behandling</span>
-                                </Flatknapp>
-                            )}
-                    </>
-                );
-            }}
-        </DataViewer>
+                            </>
+                        )}
+                    </DataViewer>
+                </tbody>
+            </table>
+            {!nyBehandling && (
+                <Flatknapp onClick={opprettBehandling}>
```

Comment:
Litt usikker på `opprettBehandling` - jeg trodde først denne opprettet en ny behandling, men ser jo at den "legger til rette for å velge en ny behandling som vil bli opprettet når du journalfører" :) 

Vet ikke om så mange bedre forslag, så får stå inntil videre, men hvis du kommer på noe så er det bare å kjøre på :)

Reply:
`lagNyBehandlingRad` kanskje?

Reply:
Ja, det liker jeg!